docs/src/getting_started.ipynb

```diff
@@ -8,7 +8,17 @@
    "source": [
     "# Getting Started with Temporian\n",
     "\n",
-    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/temporian/blob/last-release/docs/src/tutorials/getting_started.ipynb)"
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/temporian/blob/last-release/docs/src/tutorials/getting_started.ipynb)\n",
+    "\n",
+    "This guide will introduce you to the basics of Temporian, including:\n",
+    "- What is an **EventSet** and how to create one from scratch.\n",
+    "- Visualizing input/output data using **EventSet.plot()** and interactive plots.\n",
+    "- Converting back and forth between EventSets and pandas **DataFrames**.\n",
+    "- Transforming the EventSets by using **operators**.\n",
+    "- How operators work when using **indexes**.\n",
+    "- Commonly used operations like **glue**, **resample**, **lag**, moving windows and arithmetics.\n",
+    "\n",
+    "If you're interested in a topic that is not included here, we provide links to other parts of the documentation on the final section, to continue learning."
```

Comment:
I would conclude the introduction with something desirable (e.g. a carrot) about what the reader will get by reading this document. For example, something along the lines of: you will generate good features easily and without having to worry about errors. You models will perform better (maybe less cheesy).